app/views/articles/search/_meta.html.erb

```diff
@@ -4,11 +4,11 @@
 <meta name="keywords" content="software development,engineering,rails,javascript,ruby">
 
 <meta property="og:type" content="website" />
-<meta property="og:url" content="DEV => Search Results" />
-<meta property="og:title" content="The DEV Community" />
+<meta property="og:url" content="<%= ApplicationConfig['COMMUNITY_NAME'] %> => Search Results" />
```

Comment:
What if we add a `community_name` helper so we don't have to refer to an environment variable in the templates?

Reply:
Done!  😊